etc/base.yaml

```diff
@@ -182,8 +182,12 @@ management_portal:
   _chart_version: 1.1.4
   _extra_timeout: 210
   replicaCount: 1  # should be 1
+  # FIXME: Should the postgres database connection info not be a global property?
```

Comment:
Having a FIXME in this file might be confusing for the users, I think it's better to create a github issue for it

Reply:
Issue [created](https://github.com/RADAR-base/RADAR-Kubernetes/issues/269) and comment removed!